bigbluebutton-html5/imports/ui/core/graphql/queries/users.ts

```diff
@@ -14,7 +14,6 @@ subscription UserListSubscription($offset: Int!, $limit: Int!) {
                 order_by: [
                   {presenter: desc},
                   {role: asc},
-                  {raiseHandTime: asc_nulls_last},
```

Comment:
please remove this line from the index of the table:

https://github.com/bigbluebutton/bigbluebutton/blob/65e26b98035979ea1b1f9ebc46cf18cc9b517c23/bbb-graphql-server/bbb_schema.sql#L440